### ISSUE TITLE
Latest FireFox 4 Add-on and Livereload not working on OSX

### ISSUE THREAD

**Issue by hkarthik** [mockko/livereload#77]
When launching livereload from the command line, it seems to start on port 35729 and wait for a browser to connect. Any changes that I make to files are detected and showing up in the output window.

However, the FireFox 4 addon isn't able to connect. I've confirmed that Web Sockets is enabled (both the network.websocket.override-security-block and network.websocket.enabled settings are set to 'true').

The message I get in the popup is: 
    Cannot connect to LiveReload server: 
    ws://localhost:35729/websocket

Any ideas? Could it be some kind of network issue with OSX?

-Karthik

**Comment by hkarthik** [comment]
I fixed the problem. Turned out to be related to the default setting in Firefox 4 to use "System Proxy settings". I found this after noticing this in the error log.

Error: Firefox is configured to use a proxy server that is refusing connections.
Source File: chrome://livereload/content/background.js
Line: 145

A quick look at the Network Settings for Firefox 4 revealed that it was setup to use System Proxy settings. When I changed this to No Proxy, LiveReload started to work.

Hope this information helps anyone else that might be having this problem.
